nodes/rosa-managing-worker-nodes.adoc

```diff
@@ -13,9 +13,21 @@ ifdef::openshift-rosa[]
 {product-title} (ROSA).
 endif::[]
 
-
 The majority of changes for worker nodes are configured on machine pools. A _machine pool_ is a group of worker nodes in a cluster that have the same configuration, providing ease of management. You can edit the configuration of worker nodes for options such as scaling, instance type, labels, and taints.
 
+ifdef::openshift-dedicated[]
+include::modules/creating-a-machine-pool-ocm.adoc[leveloffset=+1]
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+include::modules/creating-a-machine-pool.adoc[leveloffset=+1]
+include::modules/creating-a-machine-pool-ocm.adoc[leveloffset=+2]
+include::modules/creating-a-machine-pool-cli.adoc[leveloffset=+2]
+
+.Additional resources
```

Comment:
I think the following is needed now for these?

```
[role="_additional-resources"]
.Additional resources
```

Reply:
Thank you for highlighting that. Resolved.